### PR TITLE
Adding a example project

### PR DESCRIPTION
**Pull request by odemonforte**
Hi! I'm just sharing some tests I did with Micronaut and neo4j, just for evaluation purposes.
Hope it helps!

### CODE REVIEW

File: neo4j-musicians/src/main/java/micronaut/neo4j/example/configuration/AppConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+import io.micronaut.context.annotation.Bean;
+import io.micronaut.context.annotation.Factory;
+import org.neo4j.ogm.config.Configuration;
+import org.neo4j.ogm.session.SessionFactory;
+
+@Factory
+public class AppConfiguration {
+
+    @Bean
```
Comment (jameskleeh) [inline neo4j-musicians/src/main/java/micronaut/neo4j/example/configuration/AppConfiguration.java]: `@Bean` shouldn't be used by itself. Use `@Prototype` if this should really be a prototype bean, or `@Singleton` if it should be a singleton

Comment (odemonforte) [reply neo4j-musicians/src/main/java/micronaut/neo4j/example/configuration/AppConfiguration.java]: Fixed!

Comment (odemonforte) [reply neo4j-musicians/src/main/java/micronaut/neo4j/example/configuration/AppConfiguration.java]: @jameskleeh hi! any news on this? :-)

**Comment by jameskleeh** [comment]
Thank you for the contribution!

**Pull request merged**